core/qthelper.cpp

```diff
@@ -219,6 +217,12 @@ static bool parseCoord(const QString &txt, int &pos, const QString &positives,
 		value += number / 3600.0;
 	else if (numberDefined)
 		return false;
+
+	// We parsed a valid coordinate; eat up to one separator and any
```

Comment:
I know I am nit-picking (and I can easily fix this when merging), but that comment is no longer correct...

Reply:
Ech, I fixed this and then un-fixed it by mistake